cluster-provision/gocli/opts/rookceph/manifests/cluster-test.yaml

```diff
@@ -0,0 +1,50 @@
+#################################################################################################################
+# Define the settings for the rook-ceph cluster with common settings for a small test cluster.
```

Comment:
These manifests are just copied from the existing ones?

Reply:
yes
see the comment i left on the other conversation where you mentioned this